hack/ci/build.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+# to avoid any permission problems we reset access rights recursively
+chmod -R 777 .
```

Comment:
How long does this take on an average system with a few gigabyte in place?

Reply:
So you're suggesting that we only use this on selected subdirectories in the kubevirt repository directory? Like i.e. _out and _ci-configs?

Reply:
In general yes.

Reply:
I mean, at that time we are within the kubevirt repository directory so there should just be the contents of that repo there.

Reply:
ah wait, it is the CI build file only. So no interaction with the normal developer flow.